src/modules/rlm_lua/lua.c

```diff
@@ -39,6 +39,29 @@ RCSID("$Id$")
 
 static _Thread_local REQUEST *rlm_lua_request;
 
+typedef struct radiusd_constants_t {
+	char const *name;
+	int  value;
+};
+
+static radiusd_constants_t radiusd_constants[] = {
+#define A(x) { #x, x },
+	A(RLM_MODULE_REJECT)
+	A(RLM_MODULE_FAIL)
+	A(RLM_MODULE_OK)
+	A(RLM_MODULE_HANDLED)
+	A(RLM_MODULE_INVALID)
+	A(RLM_MODULE_USERLOCK)
+	A(RLM_MODULE_NOTFOUND)
+	A(RLM_MODULE_NOOP)
+	A(RLM_MODULE_UPDATED)
+	A(RLM_MODULE_NUMCODES)
```

Comment:
- NUMCODES is not a valid constant. 
- The table they should be added to is 'fr', like the other helper functions.
- There's no point in using the full constant name.  Lua developers will have no internal knowledge of how FreeRADIUS works internally.  You should use the ``rcode_table``, and fr_str2int/fr_int2str.

The end result is that they should be able to return ``fr.ok``, ``fr.fail`` etc.. which is significantly less typing.